clodius/cli/aggregate.py

```diff
@@ -1598,18 +1537,33 @@ def bedpe(
          "should span. This determines the maximum zoom level"
 )
 @click.option(
-    '-z',
-    '--max-zoom',
-    default=19,
-    help="The number of nucleotides that the highest resolution tiles "
-         "should span. This determines the maximum zoom level"
-)
-def geojson(
-    filepath, output_file, max_per_tile, tile_size, max_zoom
-):
-    _geojson(
-        filepath, output_file, max_per_tile, tile_size, max_zoom
-    )
+    '-c',
+    '--chromsizes-filename',
+    default=None,
+    help="The file containnig chromosome sizes and order")
+@click.option(
+    '--starting-resolution',
+    '-s',
+    default=256,
+    help="The resolution that the starting data is at (e.g. 1, 10, 20)")
+@click.option(
+    '--method',
+    help='The method used to aggregate values (e.g. sum, average...)',
+    type=click.Choice(['sum', 'logsumexp']),
+    default='sum')
+@click.option(
+    '--row-infos-filename',
+    help="A file containing the names of the rows in the multivec file",
+    default=None)
+def multivec(
+        filepath, output_file, assembly, tile_size,
+        chromsizes_filename, starting_resolution, method,
+        row_infos_filename):
+    """Aggregate a multivec file"""
+    _multivec(
+        filepath, output_file, assembly, tile_size,
+        chromsizes_filename, starting_resolution, method,
+        row_infos_filename)
```

Comment:
Since the signatures are the same, this seems gratuitous... but maybe you want consistency with other private methods?

Reply:
You mean having the _multivec method in addition to the `multivec` method? I think there was a reason for this and if I recall correctly, it was that the decorated click methods were difficult to call internally. I could be wrong about that.

Reply:
That's fair enough: Not a big deal.